release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -84,7 +84,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
     1. Compare the generated output to the new commits for the release to check if
        any notable change missed a release note.
     1. Reword release notes as needed, ideally in the original PRs so that the
-       release notes can be regnerated. Make sure to check notes for breaking
+       release notes can be regenerated. Make sure to check notes for breaking
```

Comment:
Can you backport the original fix?  Especially for release-tools changes, we need to import them using git subtree.

Reply:
@xing-yang I do not have permission to put all the travis workflow related change, can someone help here ?

Error:
 (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/trivy.yaml` without `workflow` scope)

Reply:
@andyzhangx Are you available to help backport this PR https://github.com/kubernetes-csi/external-snapshotter/pull/1105 to fix the codespell issue?  Thanks!

Reply:
I have included all the required changes, and also need to fix a cve in release-8.0: https://github.com/kubernetes-csi/external-snapshotter/pull/1138